### PR TITLE
[wip] refocus READMEs at top level and core/ level

### PR DESCRIPTION
**Pull request by hpk42**
addresses #34 

- rewrite/refocus READMEs and home page
- tentatively use ``muacrypt`` as the new name

before merge all links to py-autocrypt need to replaced as well as the package
name itself etc. -- the name is not final therefore i am not doing a full walk through
with the rename.

### CODE REVIEW

**Comment by fschulze** [comment]
I'd say muacrypt is workable. It's not great, but the best of the names I heard so far.

File: doc/index.rst [hunk 1]
```diff
@@ -1,38 +1,45 @@
...
+======================================================
 
 .. note::
 
     There is a `tentative name change consideration
     <https://github.com/hpk42/py-autocrypt/issues/34>`_ for this project
-    which would result in change of links.
+    which is to result in change of links, names.
 
 py-autocrypt provides a command line tool and a Python API to help
```
Comment (azul) [inline doc/index.rst]: muacrypt instead of py-autocrypt here? Not sure which heuristic you are using for which usages to rename and which to keep.

**Comment by azul** [comment]
all in all looks good to me.

File: doc/index.rst (hunk 1 above)
Comment (hpk42) [reply doc/index.rst]: this and many other places need re-naming.  i didn't follow through because it would mean doing it all again if the name changes.

**Pull request merged**